Cargo.toml

```diff
@@ -15,8 +15,9 @@ exclude = ["doc/*", "bors.toml", ".travis.yml", "test_data/*"]
 [lib]
 
 [features]
-default = ["opengl"]
+default = ["opengl", "audio"]
 opengl = ["gfx_device_gl", "gfx_window_glutin", "glutin"]
+audio = ["rodio"]
```

Comment:
would you consider changing the feature name to simply "rodio"?

Reply:
Sure, but that sounds less like something the user may understand, at first sight.

Reply:
yeah, I take the point. Let's proceed then